app/lib/package/backend.dart

```diff
@@ -938,7 +938,7 @@ Future<_ValidatedUpload> _parseAndValidateUpload(
   final pubspec = Pubspec.fromYaml(archive.pubspecContent);
   if (!await nameTracker.accept(pubspec.name)) {
     throw GenericProcessingException(
-        'Package name is too similar to another package.');
+        'Package name is too similar to another active or moderated package.');
```

Comment:
not sure: moderated -> reserved ?

Reply:
I want to keep `moderated` as this is the term we use when we remove the package. But maybe the internal use of `reserved` is a bit confusing. I'll add a bit of documentation.

Reply:
This is an error that will show up on `pub publish` right?

As an end-user I might get confused by the term 'moderated' without further context.